templates/scripts/dl-binary.sh.j2

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# vim: ft=bash
+set -e
+
+GH_NIMBUS_RELEASE_URL="https://api.github.com/repos/status-im/nimbus-eth2/releases/latest"
+GH_NIMBUS_DOWNLOAD_URL="https://github.com/status-im/nimbus-eth2/releases/latest/download/"
+ARCH="amd64"
+BEACON_NODE_SERVICE="{{ beacon_node_service_name }}.service"
+BUILD_SERVICE="build-{{ beacon_node_service_name }}.service"
+BINARY_PATH="{{ beacon_node_bin_path }}"
+BEACON_NODE_PORT="{{ beacon_node_rest_port }}"
+#-----------------------------------------------------------------------------------
+
+function getCurrentVersion() {
+  result=$(curl -slf -X GET "http://localhost:${BEACON_NODE_PORT}/eth/v1/node/version" | jq -r '.data.version')
```

Comment:
This will not work if the node is not running or broken. You have to either parse output of `--version`, or even better, depend on the file name suffix that should contain the version.

Reply:
Updated with parsing of `--version` result.

change if by  parsing the result of ``readlink ${BINARY_PATH}/nimbus_beacon_node`` if you prefer